runtime/onert/core/include/util/TracingCtx.h

```diff
@@ -63,6 +63,13 @@ class TracingCtx
 
   uint32_t getSessionId() const { return _session_id; }
 
+  /**
+   * @brief Return true if more than 1 session exist
+   *
+   * @note  This method is NOT thread-safe. Call this in thread-safe situation.
+   */
+  bool hasMultipleSessions() const { return _next_session_id > 1; }
```

Comment:
if there are multiple session, `SNPEEventWriter` will use `"$1 session $2 subgraph $23 Add"` as op name.
However if there is only one session, `SNPEEventWriter` will use `"$2 subgraph $23 Add"` without `"$0 session"`. This is requested by Tizen people.
This method is to check if there are multiple session.